uber/site_sections/registration.py

```diff
@@ -153,6 +153,7 @@ def change_badge(self, session, id, message='', **params):
             'attendee': attendee
         }
 
+    @unrestricted
```

Comment:
By convention, instead of being in ``registration.py`` this page handler should probably be in ``preregistration.py`` which is where all the other unrestricted reg-related pages go.  Not that it really matters, since it's just a convention not a requirement.